docs/source/api/shape.rst

```diff
@@ -0,0 +1,13 @@
+.. Copyright (c) 2016, Johan Mabille, Sylvain Corlay and Wolf Vollprecht
+
+   Distributed under the terms of the BSD 3-Clause License.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+Shape/index manipulation
+========================
+
+.. toctree::
+
+   xshape
+   xstrides
```

Comment:
I think we should keep an entry for each of these files (and the corresponding sections) instead of grouping them in `Shape/Index manipulation`

Reply:
Fine. It was just aesthetically that currently all `x...` entries are folded except `xstrides`:

![xtensor](https://user-images.githubusercontent.com/7030445/226296954-f3e1575b-f7df-4f5c-860a-c026b5dea709.jpg)

My proposition would resolve this 'inconsistency'. However, if have no strong feelings for or against either.

Reply:
Oh, I missed that point. Indeed this is not consistent, you can ignore my comment ;)